rootfs/etc/cont-init.d/09-config-logrotate.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+set -e
+
+if [ ${LOGROTATE_ENABLED:-false} = true ]
```

Comment:
We should only set logrotate for the main service and not sidecar containers:

```suggestion
if [ "$SIDECAR_DISPATCHER" = "1" ] || [ "$SIDECAR_SYSLOGNG" = "1" ] || [ "$SIDECAR_SNMPTRAPD" = "1" ]; then
  exit 0
fi

if [ ${LOGROTATE_ENABLED:-false} = true ]
```

Reply:
I'm not sure for this because if we consider a main service on a host using a sidecar dispatcher on a remote host, the dispatcher will generate a lot of logs contrary to the main service.

I have tested this case, I have installed only a sidecar dispatcher on a remote host and it generates a lot of logs locally.

Do you agree that we have to keep the logrotate configuration on the sidecar_dispatcher ?